src/plugins/explore/public/components/visualizations/pie/pie_vis_options.tsx

```diff
@@ -19,54 +20,102 @@ export const PieVisStyleControls: React.FC<PieVisStyleControlsProps> = ({
   numericalColumns = [],
   categoricalColumns = [],
   dateColumns = [],
+  availableChartTypes = [],
+  selectedChartType,
+  onChartTypeChange,
 }) => {
+  // State to track expanded/collapsed state of each panel
+  const [expandedPanels, setExpandedPanels] = useState({
+    general: false,
+    basic: false,
+    exclusive: false,
+  });
+
+  const togglePanel = (panelId: string) => {
```

Comment:
nitpick, i think you can just type this as `panelId: 'general' | 'basic' | 'exclusive'` to avoid the `as keyof typeof funcof` stuff going on down there